backend/src/api/routes/places.py

```
@@ -38,14 +45,17 @@ async def get_place_photo(
                 response.iter_bytes(),
                 media_type=response.headers.get("content-type", "image/jpeg"),
                 headers={
-                    "Cache-Control": "public, max-age=86400",  # Cache for 24 hours
+                    "Cache-Control": "public, max-age=86400",
                     "Content-Length": str(len(response.content))
                 }
             )
             
     except httpx.HTTPError as e:
         logger.error(f"Failed to fetch photo from Google Places API: {e}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)